_locales/fr/messages.json

```diff
@@ -4,12 +4,12 @@
         "description": "Name of the extension."
     },
     "fullName": {
-        "message": "SponsorBlock pour YouTube - Enlève les endossements",
+        "message": "SponsorBlock pour YouTube - Enlève les messages commerciaux et publicités",
         "description": "Name of the extension."
     },
 
     "Description": {
-        "message": "Enlève les endossements dans les vidéos YouTube. Soummettre les endossements dans les vidéos que vous regardez pour aidez les autres.",
+        "message": "Passe automatiquement les messages commerciaux intégrés dans les vidéos YouTube. Soumettez les segments commerciaux dans les vidéos que vous regardez pour aidez les autres.",
```

Comment:
`Passe` was the word I was looking for, I went with `Enlève` since I forgot, thanks!

Reply:
"Enleve" is hard remove, "passe" is skip :)